src/stores/db-migrate.js

```diff
@@ -0,0 +1,22 @@
+import Umzug from 'umzug';
+import path from 'path';
+
+export async function migrate(mongoClient) {
+  const db = mongoClient.db();
+  const umzug = new Umzug({
+    storage: 'mongodb',
+    storageOptions: {
+      collection: db.collection('migrations')
+    },
+    migrations: {
+      path: path.resolve(__dirname, '../../migrations'),
+      pattern: /^\d{4}-\d{2}-\d{2}-.*\.js$/,
+      customResolver: filePath => {
+        // eslint-disable-next-line global-require
+        const Migration = require(filePath).default;
+        return new Migration(db, mongoClient);
+      }
+    }
+  });
+  await umzug.up();
+}
```

Comment:
To me it makes a bit more sense to have this file where it was, side by side with the migration scripts. But I guess there was a specific reason why it has to be moved 😬

Reply:
Yes, I moved it because now it's used by the production code to ensure the db is up to date, this used to be only some script that was ran locally. 

I can't put the migration scripts in the src. because then the build js was failing (dangling files, I guess).